packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -673,6 +682,61 @@ export enum ScalingProcess {
   ADD_TO_LOAD_BALANCER = 'AddToLoadBalancer'
 }
 
+/**
+ * EC2 Heath check options
+ */
+export interface Ec2HealthCheckOptions {
+  /**
+   * Specified the time Auto Scaling waits before checking the health status of an EC2 instance that has come into service
+   *
+   * @default -
```

Comment:
I am assuming no grace is the default?

Reply:
Yeah, like I said in the original PR, the CloudFormation doc isn't explicit, but the [API reference](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_CreateAutoScalingGroup.html) says the duration is 0 seconds.
I updated the JSDoc accordingly